docs/user-manual/index.md

```diff
@@ -351,14 +351,13 @@ support for the User Sync tool.
 ## Configuring the User Sync Tool
 
 The operation of the User Sync tool is controlled by a set of
-configuration files with these file names, located in the same
-folder as the command-line executable.
+configuration files.
 
 | Configuration File | Purpose |
 |:------|:---------|
-| dashboard&#x2011;config.yml&nbsp;&nbsp; | Required. Contains credentials and access information for calling the Adobe User Management API. |
-| connector-ldap.yml | Required. Contains credentials and access information for accessing the enterprise LDAP directory. |
-| user-sync-config.yml | Required. Contains configuration options that define the mapping of directory groups to Adobe product configurations and user groups, and that control the update behavior. |
+| user-sync-config.yml | Required. Contains configuration options that define the mapping of directory groups to Adobe product configurations and user groups, and that control the update behavior. You can specify a different configuration file using the `user-sync -c other-config.yml` command option. |
+| dashboard-config.yml | Required. Contains credentials and access information for calling the Adobe User Management API. You can specify a different dashboard configuration file, or directly specify the dashboard configuration parameters in the main configuration file. |
```

Comment:
You need to make it clear that you can specify a different configuration file for the dashboard as a setting in the main configuration file (rather than as a command-line argument) and refer them to the section on configuration file settings.  Also, don't mention specifying the dashboard parameters directly at this point, because it's an insecure option that's not recommended.